pelican/contents.py

```diff
@@ -156,13 +149,17 @@ def __init__(self, content, metadata=None, settings=None,
     def __str__(self):
         return self.source_path or repr(self)
 
-    def check_properties(self):
+    def _valid_mandatory_properties(self):
```

Comment:
Maybe `_has_mandatory_properties`, since we're checking for the existence of these properties.